### DISCUSSION TITLE
List of all semantic tokens?

### DISCUSSION THREAD

**Discussion by 0x33dm** [chakra-ui/chakra-ui#8413] [General] [answered]
I would like to know how to see a list of all semantic tokens, is there a documentation for that?

For instance i just learnt there is "ultra-slow" and "normal" for transition speed, but where can i see all those definitions?

[The documentation page ](https://chakra-ui.com/docs/styled-system/semantic-tokens)explains about them but don't mention where we can find the default values

**Comment by segunadebayo** [comment, marked as answer]
Run `npx @chakra-ui/cli eject` or read the theme source.
